### ISSUE TITLE
Snapshots panel

### ISSUE THREAD

**Issue by mbnuqw** [mbnuqw/sidebery#720]
## Description
It will show a list of snapshots and will allow user to open saved windows / restore panels with tabs / open tabs.

**Comment by sixthgear** [comment]
I'm a beta user (and former one-tab user) and this would be a fantastic feature for me. 

Snapshots (correct me if wrong) seem to be intended for users who need to "context switch" from time to time. Having quick access to these seems like a good add-on for this type of workflow.

**Comment by relink2013** [comment]
Yes please. I came here to suggest something very similar. I'd prefer a toolbar button that shows a drop down but a panel would work too. 

But I'd suggest being able to name each Window in the snapshot. I regularly have at least 2 windows open, sometimes more. I have used Simple Tabs Group for years now, but it doesn't work very well with Sidebery because it will restore all tabs into one panel. 

This is one thing Safari got right, I can name a window and it will just stay there indefinitely unless I delete it. I can restore that Window years later and everything is right where I left it.

**Comment by Dook1** [comment]
I vote for it.

**Comment by softyoda** [comment]
Hi, is there any other ways to access snapshots panel than... creating a new snapshot ? (idk where to access snapshots other than that)

**Comment by peddiashrith** [comment]
Highly valuable feature
